tests/frameworks/OSXShims/src/LoggingNative.mm

```diff
@@ -23,38 +23,38 @@ void TraceVerbose(const wchar_t* tag, const wchar_t* format, ...) {
     NSString* formatString = [NSString stringWithFormat:@"[%ws] %ws", tag, format];
     va_list varArgs;
     va_start(varArgs, format);
-    NSLog(formatString, varArgs);
+    NSLogv(formatString, varArgs);
     va_end(varArgs);
 }
 
 void TraceInfo(const wchar_t* tag, const wchar_t* format, ...) {
     NSString* formatString = [NSString stringWithFormat:@"[%ws] %ws", tag, format];
     va_list varArgs;
     va_start(varArgs, format);
-    NSLog(formatString, varArgs);
+    NSLogv(formatString, varArgs);
     va_end(varArgs);
 }
 
 void TraceWarning(const wchar_t* tag, const wchar_t* format, ...) {
     NSString* formatString = [NSString stringWithFormat:@"[%ws] %ws", tag, format];
     va_list varArgs;
     va_start(varArgs, format);
-    NSLog(formatString, varArgs);
+    NSLogv(formatString, varArgs);
     va_end(varArgs);
 }
 
 void TraceError(const wchar_t* tag, const wchar_t* format, ...) {
     NSString* formatString = [NSString stringWithFormat:@"[%ws] %ws", tag, format];
     va_list varArgs;
     va_start(varArgs, format);
-    NSLog(formatString, varArgs);
+    NSLogv(formatString, varArgs);
     va_end(varArgs);
 }
 
 void TraceCritical(const wchar_t* tag, const wchar_t* format, ...) {
     NSString* formatString = [NSString stringWithFormat:@"[%ws] %ws", tag, format];
```

Comment:
I can't find a reference to `%ws` in the IEEE printf specification; should this be `%ls` instead?